src/main/kotlin/no/nav/sifinnsynapi/konsumenter/K9Beskjed.kt

```diff
@@ -38,14 +49,15 @@ fun K9Beskjed.somNøkkel(systembruker: String): Nokkel {
 }
 
 fun K9Beskjed.somBeskjed(): Beskjed {
-    val linkUtenNull = link ?: ""
-    return Beskjed(
-        System.currentTimeMillis(),
-        Instant.now().plus(dagerSynlig, ChronoUnit.DAYS).toEpochMilli(),
-        søkerFødselsnummer,
-        grupperingsId,
-        tekst,
-        linkUtenNull,
-        4
-    )
+    val builder = BeskjedBuilder()
+        .withEksternVarsling(false)
+        .withFodselsnummer(this.søkerFødselsnummer)
+        .withGrupperingsId(this.grupperingsId)
+        .withSikkerhetsnivaa(4)
+        .withSynligFremTil(LocalDateTime.now().plusDays(dagerSynlig))
+        .withTekst(tekst)
+        .withTidspunkt(LocalDateTime.now(UTC))
+
+    if(link != null) builder.withLink(URL(link))
```

Comment:
Kan forenkles med `link?.let { builder.withLink(URL(it)) }` om du skulle ønske å bruke det :)